src/app/api/youtube/models.ts

```
@@ -0,0 +1,39 @@
+
+// TODO: Consider checking if a playlist has 0 items, hence don't take that into account
+//  "contentDetails": {
+// "itemCount": 0
+// }
+
+// Get all playlists by channel
+export async function getPlaylistsByChannel(channelId: string) {
+  const response = await fetch(YOUTUBE_CHANNEL_PLAYLISTS(channelId));
+  const firstPageData = await response.json();
+  if (response.status === 200) {
+    let playlistItems = firstPageData.items;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'playlistItems' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)